client/reader/following-manage/style.scss

```diff
@@ -202,6 +202,17 @@
 	}
 }
 
+.following-manage__search-results.is-empty {
+	color: darken( $gray, 10% );
+}
+
+.following-manage__subscriptions-list.is-empty {
+	color: darken( $gray, 10% );
+	p {
```

Comment:
We should target this `p` directly with a class name rather than styling the element.